assets/src/dashboard/components/button/index.js

```diff
@@ -26,14 +26,18 @@ import styled from 'styled-components';
 import { BUTTON_TYPES, KEYBOARD_USER_SELECTOR } from '../../constants';
 
 const StyledButton = styled.button`
+  font-family: ${({ theme }) => theme.fonts.button.family};
+  font-size: ${({ theme }) => theme.fonts.button.size};
+  font-weight: ${({ theme }) => theme.fonts.button.weight};
+  line-height: ${({ theme }) => theme.fonts.button.lineHeight};
```

Comment:
I moved the font controls to `StyledButton` so they can be overridden using a styled component.